packages/h/hyprland/package.yml

```diff
@@ -0,0 +1,47 @@
+name       : hyprland
+version    : 0.43.0
+release    : 1
+source     :
+    - https://github.com/hyprwm/Hyprland/releases/download/v0.43.0/source-v0.43.0.tar.gz : fa2ce308f6f4524d1564163fd5ada2ccac919b4b48048b07f04e6b96aa0f1e6c
+homepage   : https://hyprland.org/
+license    : BSD-3-Clause
+component  : desktop.hyprland
+summary    : Hyprland is a 100% independent, dynamic tiling Wayland compositor that doesn't sacrifice on its looks.
+description: |
+    Hyprland is a 100% independent, dynamic tiling Wayland compositor that doesn't sacrifice on its looks. It provides the latest Wayland features, is highly customizable, has all the eyecandy, the most powerful plugins, easy IPC, much more QoL stuff than other compositors and more...
+builddeps  :
+    - pkgconfig(aquamarine)
+    - pkgconfig(cairo)
+    - pkgconfig(gbm)
+    - pkgconfig(gio-2.0)
+    - pkgconfig(hwdata)
+    - pkgconfig(hyprcursor)
+    - pkgconfig(hyprlang)
+    - pkgconfig(hyprutils)
+    - pkgconfig(hyprwayland-scanner)
+    - pkgconfig(libdisplay-info)
+    - pkgconfig(libdrm)
+    - pkgconfig(libinput)
+    - pkgconfig(libliftoff)
+    - pkgconfig(libseat)
+    - pkgconfig(libudev)
+    - pkgconfig(pango)
+    - pkgconfig(pangocairo)
+    - pkgconfig(pixman-1)
+    - pkgconfig(tomlplusplus)
+    - pkgconfig(uuid)
+    - pkgconfig(wayland-client)
+    - pkgconfig(wayland-cursor)
+    - pkgconfig(wayland-protocols)
+    - pkgconfig(wayland-server)
+    - pkgconfig(xcb-errors)
+    - pkgconfig(xcb-ewmh)
+    - pkgconfig(xcb-icccm)
+    - pkgconfig(xcb-util)
+    - pkgconfig(xcursor)
+    - pkgconfig(xkbcommon)
+    - pkgconfig(xwayland)
+build      : |
+    %make
+install    : |
+    %make_install
```

Comment:
I see that they have meson and cmake build files. Is there something wrong with using one of those?

Reply:
the official website recommands to use this command "make all && sudo make install", i tried to make the same in the package file. Therefore meson build ends with errors